installing/installing_cloud_other/verify-installation-cloud-other.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="verify-installation-cloud-other"]
+= Verifying installation of RHACS Cloud Service on other platforms
+include::modules/common-attributes.adoc[]
+:context: verify-installation-cloud-other
+
+toc::[]
+
+[role="_abstract"]
+Provides steps to verify that {product-title-managed-short} is properly installed.
+
+:FeatureName: {product-title-managed-short}
+include::snippets/field-trial.adoc[leveloffset=+1]
+
+To verify installation, access your ACS Console from the {cloud-console}. If the system is operating correctly, the Dashboard will display the number of clusters that {product-title-managed-short} is monitoring, along with information about nodes, deployments, images, and violations.
```

Comment:
Do we need _If the system is operating correctly,_?

_The Dashboard should display..._